openshift/kustomize/services/contentmigration-historic/base/config-map.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Couldn't you have a different overlay instead of another config?

Reply:
Funny you should say that because I was thinking the same thing, but when I talked to Kulpreet about this he seemed to think this was easier.